tests/Unit/HC/multi_acc2.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %hc --amdgpu-target=gfx701 --amdgpu-target=gfx801 --amdgpu-target=gfx802 --amdgpu-target=gfx803 %s -o %t.out && %t.out
```

Comment:
maybe we should remove the static list of gpu and just use the enumerator?

Reply:
As mentioned in my previous comment I'd like to keep `hcc` be less dependent to `rocm_agent_enumerator`.